helloworld.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello World</title>
+    <link rel="stylesheet" href="hello.css">
+
+  </head>
+
+  <body>
+    <div class="header">
```

Comment:
- [ ] For the semantic reasons I will suggest you change the class header name for good practice :+1: